packages/sanity/src/structure/panes/documentList/DocumentSheetListPane.tsx

```diff
@@ -82,6 +84,13 @@ function DocumentSheetListPaneInner({
     autoResetPageIndex: false,
     initialState: {
       pagination: {pageSize: 25},
+      columnVisibility: initialColumnsVisibility,
+    },
+    meta: {
+      updateData: () => {
```

Comment:
What does this do? Can you add a comment on why this `console.log` is okay here?